package.json

```diff
@@ -80,6 +80,7 @@
     "qs": "^6.5.1",
     "query-string": "^4.3.4",
     "ramda": "^0.24.1",
+    "raven-js": "^3.19.1",
```

Comment:
This is the only reference to `raven-js` in the entire PR... I'd imagine you'd have to at least 
```javascript
import Raven from 'raven-js';
window.Raven = Raven;
```
or something...